HCDevice.py

```diff
@@ -129,25 +173,32 @@ def get(self, resource, version=1, action="GET", data=None):
 		}
 
 		if data is not None:
-			msg["data"] = [data]
+			if action == "POST":
+				if self.test_feature(data) != True:
```

Comment:
E712 comparison to True should be 'if cond is not True:' or 'if not cond:'
 
 Or is there a reason for this implementation?